Tribler/Test/Core/TorrentChecker/test_torrentchecker_session.py

```diff
@@ -275,6 +275,17 @@ def on_error(failure):
 
         return test_deferred
 
+    import mock
+
+    def dummy_is_pending_task_active(*args, **kwargs):
+        return True
+
+    @mock.patch("Tribler.Core.TorrentChecker.session.UdpTrackerSession.is_pending_task_active",
```

Comment:
Don't use the `mock` framework; take a look at the other tests in the same file to see how you can do it without the specified library.